project/ws/app/src/lib/routes/profile-v2/components/transfer-request/transfer-request.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="flex flex-col gap-4">
+    <div class="text-base font-semibold text-center">Select all the fields to make transfer request.</div>
+
+    <form class="flex flex-col gap-4" [formGroup]="transferRequestForm">
+        <div class="flex flex-col">
+            <div class="text-sm font-bold required">Organization</div>
```

Comment:
Hardcoded text should come from language files

Reply:
I will do it everything at last, when I complete functionality